pack/tic/itd_encounter.json

```diff
@@ -1,19 +1,531 @@
+		"enemy_damage": 2,
+		"enemy_evade": 1,
+		"enemy_fight": 2,
+		"enemy_horror": 1,
+		"faction_code": "mythos",
+		"health": 4,
+		"illustrator": "Matthew Cowdery",
+		"name": "Ravager from the Deep",
+		"pack_code": "itd",
+		"position": 145,
+		"quantity": 2,
+		"text": "Hunter.\nWhile its location is flooded, Ravager from the Deep gets +1 fight and +1 evade (+2 fight and +2 evade instead if its location is fully flooded).\n<b>Forced</b> - After Ravager from the Deep engages you: Test [combat] (2). If you fail, place 1 barrier between your location and the connecting location to the west.",
+		"traits": "Humanoid. Monster. Deep One.",
+		"type_code": "enemy"
+	},
+	{
+		"code": "07146",
+		"encounter_code": "in_too_deep",
+		"encounter_position": 25,
+		"enemy_damage": 1,
+		"enemy_evade": 1,
+		"enemy_fight": 3,
+		"enemy_horror": 1,
+		"faction_code": "mythos",
+		"health": 2,
+		"illustrator": "Mike Capprotti",
+		"name": "Emerging Deep One",
+		"pack_code": "itd",
+		"position": 146,
+		"quantity": 3,
+		"text": "Hunter.\n<b>Revelation</b> - Emerging Deep One enters play exhausted and unengaged unless its location is fully flooded.\n<b>Forced</b> - After Emerging Deep One engages you, if your location is flooded: It attacks you.",
+		"traits": "Humanoid. Monster. Deep One.",
+		"type_code": "enemy"
+	},
+	{
+		"code": "07147",
+		"encounter_code": "in_too_deep",
+		"encounter_position": 28,
+		"faction_code": "mythos",
+		"illustrator": "Tomasz Jedruszek",
+		"name": "Deep One Invasion",
+		"pack_code": "itd",
+		"position": 147,
+		"quantity": 1,
+		"text": "<b>Revelation</b> - Shuffle the encounter discard pile into the encounter deck. For each location to the east of your location (on the same row), discard cards from the top of the encounter deck until a [[Deep One]] enemy with the hunter keyword is discarded and spawn that enemy at that location.",
+		"traits": "Scheme.",
+		"type_code": "treachery"
+	},
+	{
+		"code": "07148",
+		"encounter_code": "in_too_deep",
+		"encounter_position": 29,
+		"faction_code": "mythos",
+		"illustrator": "Cyril van der Haegen",
+		"name": "Pulled Back",
+		"pack_code": "itd",
+		"position": 148,
+		"quantity": 2,
+		"text": "<b>Revelation</b> - Test [willpower] (3). If you fail, move to the connecting location to the east, ignoring all barriers. Then place each key you control on your location.",
+		"traits": "Terror.",
+		"type_code": "treachery"
+	},
+	{
+		"code": "07149",
+		"encounter_code": "in_too_deep",
+		"encounter_position": 31,
+		"faction_code": "mythos",
+		"illustrator": "Lukasz Jaskolski",
+		"name": "Inundated",
+		"pack_code": "itd",
+		"position": 149,
+		"quantity": 3,
+		"text": "<b>Revelation</b> - Place 1 barrier between this location and each connecting location with no barriers between them. If no barriers are placed by this effect, Inundated gains surge.",
+		"traits": "Hazard.",
+		"type_code": "treachery"
+	},
+	{
+		"code": "07150",
+		"cost": 0,
+		"deck_limit": 1,
```

Comment:
please remove this line

Reply:
Done. Just want to mention that I used "Thomas Dawson" from tic_encounter.json as reference when I decided to include this one. So it might need to be removed for that one as well?

Reply:
Yeah, Thomas Dawson has a Player Card back, while this one has an Encounter Card back (which I don't think we can tell from CardGameDB directly, have to see the real cards).

Reply:
Ah, understood, thanks.